.zenodo.json

```diff
@@ -4,6 +4,946 @@
       "affiliation": "Kitware, Inc",
       "name": "McCormick, Matthew",
       "orcid": "0000-0001-9475-3756"
+    },
+    {
+      "affiliation": "University of Iowa",
+      "name": "Johnson, Hans",
+      "orcid": "0000-0001-9513-2660"
+    },
+    {
+      "affiliation": "Universit\u00e9 de Sherbrooke",
+      "name": "Legarreta, Jon Haitz",
+      "orcid": "0000-0002-9661-1396"
+    },
+    {
+      "name" : "Luis Ibanez"
+    },
+    {
+      "name" : "Bill Lorensen"
+    },
+    {
+      "name" : "Brad King"
+    },
+    {
+      "name" : "Daniel Blezek"
+    },
+    {
+      "name" : "Bradley Lowekamp"
+    },
+    {
+      "name" : "Julien Jomier"
+    },
+    {
+      "name" : "Jim Miller"
+    },
+    {
+      "name" : "Gaëtan Lehmann"
+    },
+    {
+      "name" : "Josh Cates"
+    },
+    {
+      "name" : "Lydia Ng"
+    },
+    {
+      "name" : "Jisung Kim"
+    },
+    {
+      "name" : "Arnaud Gelas"
+    },
+    {
+      "name" : "Mathieu Malaterre"
+    },
+    {
+      "name" : "Karthik Krishnan"
+    },
+    {
+      "name" : "Bill Hoffman"
+    },
+    {
+      "name" : "Kent Williams"
+    },
+    {
+      "name" : "Francois Budin"
+    },
+    {
+      "name" : "Will Schroeder"
+    },
+    {
+      "name" : "Stephen R. Aylward"
+    },
+    {
+      "name" : "Dženan Zukić"
```

Comment:
Is now the right time to add ORCIDs (mine is 0000-0002-9546-9703), or in follow-up PRs?

Reply:
I think they should be added now, and merged soon.

Reply:
> I think they should be added now, and merged soon.

@hjmjohnson could you please push the id's you provided below to the branch?